lib/binary_to_decimal.rb

```diff
@@ -5,5 +5,30 @@
 # Calculate  and return the decimal value for this binary number using
 # the algorithm you devised in class.
 def binary_to_decimal(binary_array)
```

Comment:
This works, but it doesn't scale well.  

Could you do it with a loop which goes from `binary_array.length - 1` down to 0.

```ruby
index = binary_array.length - 1
value = 0
while index >= 0
  value += binary_array[index] * 2 ** index
  index -= 1
end
return value
```